charts/hub-agent/values.yaml

```diff
@@ -32,5 +32,12 @@ tolerations: []
 
 affinity: {}
 
-enableV1Alpha1APIs: true
-enableV1Beta1APIs: false
+enableV1Alpha1APIs: false
+enableV1Beta1APIs: true
```

Comment:
Hi Britania! This will lead to failure in our E2E test for the v1alpha1 version; would you mind updating the Makefile so that the v1alpha1 version uses the old API? Thanks!